src/components/Home.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react'
+import ReactPaginate from 'react-paginate';
+import Table from './Table'
+import { axiosGet } from './Api';
+import Filters from './Filters';
+
+
+const Home = () => {
+    const [beerName, setbeerName] = useState("")
+    const [yeast, setYeast] = useState("");
+    const [date, setDate] = useState("");
+    const [abv, setAbv] = useState("");
+    const [id, setId] = useState("");
+    const [pageNo, setpageNo] = useState(33)
+    const [noOfPost, setNoOfPost] = useState(10)
+    const [pageNext, setNextPage] = useState(1)
+    const [post, setpost] = useState([])
```

Comment:
Rather than using multiple states for each filter find a better and compact solution for this.